rp/src/exchange.rs

```diff
@@ -271,6 +309,7 @@ pub async fn exchange(options: ExchangeOptions) -> Result<()> {
         anyhow::Error::msg(format!("NativeUnixBrokerConfigBaseBuilderError: {:?}", e))
     }
 
+    // configure everything per peer
```

Comment:
A style thing, should we always start comments uppercase or keep it random? CC @koraa

Reply:
Is there a consistent policy in the other crates? If so, I'm happy to follow that policy as well.

Either way, it shouldn't be too much of an issue for me to change the hand full of comments to match what is done in the other crates.